lib/HLSL/DxilContainerReflection.cpp

```diff
@@ -2428,7 +2430,8 @@ HRESULT DxilShaderReflection::GetInputParameterDesc(UINT ParameterIndex,
   else
     memcpy(pDesc, &m_InputSignature[ParameterIndex],
            // D3D11_43 does not have MinPrecison.
-           sizeof(D3D12_SIGNATURE_PARAMETER_DESC) - sizeof(D3D_MIN_PRECISION));
+           offsetof(D3D12_SIGNATURE_PARAMETER_DESC, Stream) +
+               sizeof(D3D12_SIGNATURE_PARAMETER_DESC::Stream));
```

Comment:
I'm guessing the simpler approach of using `offsetof(D3D12_SIGNATURE_PARAMETER_DESC, MinPrecision)` for the size fails because of alignment issues?

Reply:
Right, happens to be equivalent in this case, but generally, if there was any padding between these fields, we wouldn't want to write into that.  So, I feel like the structure of this statement more accurately reflects the intent (only write to the end of the Stream field).